read-weather.mjs

```diff
@@ -0,0 +1,30 @@
+import { readFile } from 'fs';
+import { promisify } from 'util';
+
+const readFileAsync = promisify(readFile);
+
+export const readWeatherFile = async (filePath) => {

```

Comment:
Make this and the other functions accept an object like this:
`export const readWeatherFile = async ({ filePath }) => {`